### ISSUE TITLE
Crashed cloning environment.

### ISSUE THREAD

**Issue by flamingbear** [conda/conda#2278]
Just reporting a crash while cloning environment.

```
(py3)savoie@savoie-laptop ~
 $ conda create --name ipython3 --clone py3
Fetching package metadata: ........
src_prefix: '/Users/savoie/anaconda/envs/py3'
dst_prefix: '/Users/savoie/anaconda/envs/ipython3'
Packages: 173
Files: 1105
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/savoie/anaconda/envs/py3/bin/conda", line 6, in <module>
    sys.exit(main())
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 139, in main
    args_func(args, p)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 146, in args_func
    args.func(args, p)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 49, in execute
    install.install(args, parser, 'create')
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 247, in install
    clone(args.clone, prefix, json=args.json, quiet=args.quiet, index=index)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 84, in clone
    quiet=quiet, index=index)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/misc.py", line 246, in clone_env
    sorted_dists = r.dependency_sort(dists)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/resolve.py", line 729, in dependency_sort
    depends = lookup(value)
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/resolve.py", line 724, in lookup
    return set(ms.name for ms in self.ms_depends(value + '.tar.bz2'))
  File "/Users/savoie/anaconda/lib/python2.7/site-packages/conda/resolve.py", line 573, in ms_depends
    deps = [MatchSpec(d) for d in self.index[fn].get('depends', [])]
KeyError: 'nasateam-0.1.1-py35_0.tar.bz2'
```

```
(py3)savoie@savoie-laptop ~
 $ conda info
Current conda install:

             platform : osx-64
        conda version : 4.0.4
  conda-build version : 1.19.2
       python version : 2.7.11.final.0
     requests version : 2.9.1
     root environment : /Users/savoie/anaconda  (writable)
  default environment : /Users/savoie/anaconda/envs/py3
     envs directories : /Users/savoie/anaconda/envs
        package cache : /Users/savoie/anaconda/pkgs
         channel URLs : https://conda.anaconda.org/nsidc/osx-64/
                        https://conda.anaconda.org/nsidc/noarch/
                        https://conda.anaconda.org/nsidc/channel/dev/osx-64/
                        https://conda.anaconda.org/nsidc/channel/dev/noarch/
                        https://repo.continuum.io/pkgs/free/osx-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/osx-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /Users/savoie/.condarc
    is foreign system : False
```

**Comment by kalefranz** [comment]
Hello, Could you also provide the output of `conda list --explicit -n py3`?  Will help reproduce...

**Comment by flamingbear** [comment]
That will probably help reproduce it.  
`nasateam` is a package I created and might/probably have installed from a local tarball.


```
(sea_ice_tools)savoie@savoie-laptop ~/projects/seaice/sea_ice_tools/source (daily_extent)
 $ conda list --explicit -n py3
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: osx-64
@EXPLICIT
https://repo.continuum.io/pkgs/free/osx-64/abstract-rendering-0.5.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/alabaster-0.7.7-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/anaconda-2.5.0-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/anaconda-client-1.2.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/appnope-0.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/appscript-1.0.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/argcomplete-1.0.0-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/astroid-1.3.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/astropy-1.1.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/babel-2.2.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/basemap-1.0.7-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/beautifulsoup4-4.4.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/bitarray-0.8.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/blaze-core-0.9.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/bokeh-0.11.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/boto-2.39.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/bottleneck-1.0.0-np110py35_0.tar.bz2
https://conda.anaconda.org/nsidc/osx-64/bumpversion-0.5.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cffi-1.2.1-py35_0.tar.bz2
https://conda.anaconda.org/nsidc/osx-64/click-5.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cligj-0.2.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/clyent-1.2.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/colorama-0.3.6-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/configobj-5.0.6-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cryptography-1.0.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/curl-7.45.0-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cycler-0.9.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cython-0.23.4-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/cytoolz-0.7.5-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/datashape-0.5.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/decorator-4.0.6-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/docutils-0.12-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/dynd-python-0.7.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/et_xmlfile-1.0.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/fastcache-1.0.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/fiona-1.6.0-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/flake8-2.3.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/flask-0.10.1-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/freetype-2.5.5-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/futures-3.0.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/gdal-2.0.0-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/geos-3.3.3-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/greenlet-0.4.9-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/h5py-2.5.0-np110py35_4.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/hdf5-1.8.15.1-2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/idna-2.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipykernel-4.2.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipython-4.0.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipython-notebook-4.0.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipython-qtconsole-4.0.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipython_genutils-0.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ipywidgets-4.1.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/itsdangerous-0.24-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jbig-2.1-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jdcal-1.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jinja2-2.8-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jpeg-8d-1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jsonschema-2.4.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jupyter-1.0.0-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jupyter_client-4.1.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jupyter_console-4.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jupyter_core-4.0.6-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/krb5-1.13.2-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/launcher-1.0.0-3.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libdynd-0.7.1-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libgdal-2.0.0-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libnetcdf-4.3.3.1-1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libpng-1.6.17-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libtiff-4.0.6-1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libxml2-2.9.2-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/libxslt-1.1.28-2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/llvmlite-0.8.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/logilab-common-0.63.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/lxml-3.5.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/markupsafe-0.23-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/matplotlib-1.5.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/mccabe-0.3-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/mistune-0.7.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/mkl-11.3.1-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/mkl-service-1.1.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/multipledispatch-0.4.8-py35_0.tar.bz2
# no URL for: nasateam-0.1.1-py35_0
https://repo.continuum.io/pkgs/free/osx-64/nbconvert-4.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/nbformat-4.0.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/netcdf4-1.1.9-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/networkx-1.11-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/nltk-3.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/node-webkit-0.10.1-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/nose-1.3.7-py35_0.tar.bz2
https://conda.anaconda.org/nsidc/osx-64/nose-exclude-0.4.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/notebook-4.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/numba-0.23.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/numexpr-2.4.6-np110py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/numpy-1.10.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/odo-0.4.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/openpyxl-2.3.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2f-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pandas-0.17.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/path.py-8.1.2-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/patsy-0.4.0-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pep8-1.7.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pexpect-3.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pickleshare-0.5-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pillow-3.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pip-8.0.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ply-3.8-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/psutil-3.4.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ptyprocess-0.5-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/py-1.4.31-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyasn1-0.1.9-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pycosat-0.6.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pycparser-2.14-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pycrypto-2.6.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pycurl-7.19.5.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyflakes-1.0.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pygments-2.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pylint-1.4.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyopenssl-0.15.1-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyparsing-2.0.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyqt-4.11.4-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pytables-3.2.2-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pytest-2.8.5-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/python-3.5.1-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/python-dateutil-2.4.2-py35_0.tar.bz2
https://conda.anaconda.org/nsidc/osx-64/python-termstyle-0.1.10-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/python.app-1.2-py35_4.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pytz-2015.7-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyyaml-3.11-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyzmq-15.2.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/qt-4.8.7-1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/qtconsole-4.1.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/readline-6.2-2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/redis-2.6.9-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/redis-py-2.10.3-py35_0.tar.bz2
https://conda.anaconda.org/nsidc/osx-64/rednose-0.4.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/requests-2.9.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/scikit-image-0.11.3-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/scikit-learn-0.17-np110py35_2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/scipy-0.17.0-np110py35_0.tar.bz2
https://conda.anaconda.org/nsidc/channel/dev/osx-64/seaicedata-3.0.1dev-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/setuptools-20.2.2-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/shapely-1.5.11-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/simplegeneric-0.8.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sip-4.16.9-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/six-1.10.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/snowballstemmer-1.2.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sockjs-tornado-1.0.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sphinx-1.3.5-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sphinx_rtd_theme-0.1.9-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/spyder-2.3.8-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/spyder-app-2.3.8-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sqlalchemy-1.0.11-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sqlite-3.9.2-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/statsmodels-0.6.1-np110py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sympy-0.7.6.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/terminado-0.5-py35_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/tk-8.5.18-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/toolz-0.7.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/tornado-4.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/traitlets-4.1.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ujson-1.33-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/unicodecsv-0.14.1-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/werkzeug-0.11.3-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/wheel-0.29.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/xlrd-0.9.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/xlsxwriter-0.8.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/xlwings-0.6.4-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/xlwt-1.0.0-py35_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/xz-5.0.5-1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/yaml-0.1.6-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/zeromq-4.1.3-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/zlib-1.2.8-0.tar.bz2
```

**Comment by kalefranz** [comment]
Local tarball installs have been fixed in conda 4.2 and beyond.
